opam-ci-check/lib/revdeps.ml

```diff
@@ -1,95 +0,0 @@
-(* SPDX-License-Identifier: Apache-2.0
```

Comment:
Do we really want to rm all of this code at this point?

Reply:
Sure, we could leave it in. I removed it when our thinking was to use the obuilder-spec -> docker build route.